src/eip4824.ts

```diff
@@ -34,31 +34,68 @@ router.get('/:space', async (req, res) => {
 });
 
 router.get('/:space/members', async (req, res) => {
-  let space: any = {};
-
   try {
-    space = await getSpace(req.params.space);
+    const spaceId = req.params.space;
+    const cursor = req.query.cursor || null;  
+    const pageSize = 500; // Default page size
+
+    const space = await getSpace(spaceId);
+    if (!space.verified) {
+      return res.status(400).json({
+        error: 'INVALID_SPACE',
+        message: 'The specified space is not verified.'
+      });
+    }
 
-    if (!space.verified) return res.status(400).json({ error: 'INVALID' });
+    const combinedMembersResult = await getCombinedMembersAndVoters(
+      spaceId,
+      cursor,
+      pageSize,
+      space.admins || [],
+      space.moderators || [],
+      space.members || []
+    );
+
+    const members = [
+      ...space.admins.map(admin => ({ type: 'EthereumAddress', id: admin })),
+      ...space.moderators.map(moderator => ({ type: 'EthereumAddress', id: moderator })),
+      ...space.members.map(member => ({ type: 'EthereumAddress', id: member })),
+      ...combinedMembersResult.members.map(voter => ({ type: 'EthereumAddress', id: voter }))
+    ];
```

Comment:
I feel that irrespective of `cursor` input, this will always return admins, moderators, and members on top

Reply:
@Rashmi-278 Could you please check this?

Reply:
using cursor we should exclude the members

Reply:
This improvement is added.